src/components/Corpus.jsx

```diff
@@ -3,17 +3,70 @@ import Radium from 'radium'
 
 import Styles from './Styles'
 import Strings from './Strings'
+import TitleCard from './TitleCard'
 
 @Radium
-export default class Word extends React.Component {
+export default class Corpus extends React.Component {
+  delete = () => {
+      this.props.delete(this.props.corpus)
```

Comment:
this is fine, but be more specific and consistent and change this to deleteCoprus also